examples/robolectricExample/build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'de.undercouch.download'
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.application'
+apply plugin: 'android-command'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'realm-android'
+apply plugin: 'realm-robolectric'
+
+android {
+    compileSdkVersion rootProject.sdkVersion
+    buildToolsVersion rootProject.buildTools
+
+    defaultConfig {
+        applicationId 'io.realm.examples.robolectric'
+        targetSdkVersion 21
+        minSdkVersion 15
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+
+    command {
+        events 2000
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:23.1.0'
+    compile 'com.android.support:design:23.1.0'
+    testCompile 'io.reactivex:rxjava:1.1.0'
+    testCompile 'junit:junit:4.12'
+    testCompile "org.robolectric:robolectric:3.0"
+
+    testApt "io.realm:realm-annotations-processor:${currentVersion}"
```

Comment:
Yes, I think `realm-android` would be the correct place. Because after this is merged we should also look into how easy it would be to support ordinary Android unit tests running on the JVM.